tests/python/relay/test_pass_manager.py

```diff
@@ -536,10 +537,12 @@ def test_print_ir(capfd):
 __TRACE_COUNTER__ = 0
 
 
-def _tracer(module, info, is_before):
-    global __TRACE_COUNTER__
-    if bool(is_before):
+@tvm.instrument.pass_instrument
+class MyInstrument:
+    def run_before_pass(self, module, info):
+        global __TRACE_COUNTER__
```

Comment:
Can you switch this test to not use a global.

Reply:
Done